### PR TITLE
8queen

### PR DESCRIPTION
**Pull request by kasperp**
This 

* Adds 8queen demo application
* Update iHear and basics to work with API changes regarding phases
* Adds common data packages Ace Model XML Export and VT Package

### CODE REVIEW

File: ihear/src/api/index.ts [hunk 1]
```diff
@@ -112,5 +111,6 @@ export const getBrochureModels = async (packagePath: string) => {
 };
 
 const getBrochureModelVariable = (resp: ResponseType) =>
-  resp.sections[0].variables.find((v) => v.id === BROCHURE_MODEL_VARIABLE_ID) ||
-  ({} as Variable);
+  resp.phases[0].sections[0].variables.find(
```
Comment (kasperp) [inline ihear/src/api/index.ts:114]: The brochure model variable is part of the phases area.

File: ihear/src/api/index.ts [hunk 2]
```diff
@@ -28,9 +29,9 @@ const CONFIGURE_URL = ``;
 
 const baseRequest: RequestType = {
   currency: 'EUR',
-  language: 'uk_en',
+  language: 'EN_UK',
   date: new Date(),
-  viewId: 'DEFAULT',
+  viewId: 'DEMO',
```
Comment (kasperp) [inline ihear/src/api/index.ts:34]: The Model now as a view called DEMO

File: basics/src/pages/configurator/index.js [hunk 3]
```diff
@@ -101,17 +101,20 @@ class Configurator extends React.Component {
...
             variableId: a.variable.id,
             value: a.value.value,
-            exclude: a.value.exclude
-          }))
-        }
+            exclude: a.value.exclude,
+          })),
+        },
+        settings: {
+          phaseBehavior: 'InSections',
```
Comment (kasperp) [inline basics/src/pages/configurator/index.js:116]: For now, the basics example uses the legacy setting for phases. 

I'll introduce better support for phases I'll change this.

File: 8queen/.env [hunk 4]
```diff
@@ -0,0 +1,2 @@
+REACT_APP_API_URL="https://daily.clm-dev.cloud/configurator/v1/configure"
```
Comment (mortenab) [inline 8queen/.env:1]: Not sure that we want to advertise this URL to the outside world. If we include a URL in our samples, I think it should be a more official one that partners also have access to.

One solution could be that we deploy a separate platform installation for running demos and sample applications - using our current domain something like: `http://demo.clm-dev.cloud`. Every time we release, we update that instance to the latest version and make sure the samples work.

It should only take me an hour or two to deploy a new one. We might even put the API key in the sample code?

File: 8queen/README.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+And start the application with:
+
+```
+npm start
+```
+
+## Deployment to vercel
+
+After having logged in to Vercel run
```
Comment (mortenab) [inline 8queen/README.md:29]: Could maybe explain what Vercel is and when you would use it...

**Review by mortenab — APPROVED** [review]
Nice 👍

File: 8queen/.env (hunk 4 above)
Comment (kasperp) [reply 8queen/.env:1]: That would be great if we could do that 👍

File: 8queen/README.md (hunk 5 above)
Comment (kasperp) [reply 8queen/README.md:29]: Yeah, good point. I'll delete this section and add auto vercel deployment to Github.

File: 8queen/.env (hunk 4 above)
Comment (mortenab) [reply 8queen/.env:1]: Ok - will look into that next week. Will add a Jira for it.

**Pull request merged**